### DISCUSSION TITLE
Mods in multi lobby will not deselect unless map, or diff is changed

### DISCUSSION THREAD

**Discussion by FishBulbish** [ppy/osu#14326] [Q&A] [answered]
When forcing mods on all players in a multi lobby it is impossible to disable said mods without changing the song, or difficulty after altering selections in the mods menu.

**Comment by peppy** [comment]
Could you please provide a video showing what you are seeing? I've tested and can't reproduce the issue.

**Comment by FishBulbish** [comment, marked as answer]
Thank you for looking into the issue, however I seem to be unable to
replicate it now. The bug may have been caused by my client
or server connection (I doubt this because various other players were
having the same issue). Sorry for wasting your time.
